src/vec_runtime_PWR9.c

```diff
@@ -21,11 +21,15 @@
  */
 
 #if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#ifdef __clang__
+// hack clang does not support pragma GCC target
+#else
 /* Older distros running Big Endian are unlikely to support PWR9.
  * So build PWR9 runtime for LE only.  */
 #pragma GCC target ("cpu=power9")
 
 #include "vec_int512_runtime.c"
 #endif
+#endif
```

Comment:
the comment refers to #pragma, but you also skip vec_int512_runtime.c. Is that intentional? If so, it could use a comment about what's going on.

Reply:
Not need anymore with ifunc-autoconf change, removed.